packages/patternfly-4/react-inline-edit-extension/src/components/InlineEdit/editableTableBody.d.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, FunctionComponent, MouseEvent } from 'react';
+import { TableBodyProps } from '@patternfly/react-table';
+
+import { OneOf } from '../../../../react-core/src/typeUtils';
```

Comment:
is it really ok to reference the files in different module? The same thing occurs in `Table.d.ts`

Reply:
my preference would be to export these in `react-core` like we did in #1232. Seems they are helpful in other packages too.

Reply:
done, can you please take a look at https://github.com/patternfly/patternfly-react/pull/1521?